10_2/ex04-02.js

```diff
@@ -0,0 +1,36 @@
+const messageDelay = () => Math.floor(Math.random() * 5000);
+
+const getMarsTemperature = () => {
+  const maxTemperature = 58;
+  return Math.floor(Math.random() * maxTemperature);
+};
+
+const toFahrenheit = degreeCelsius => (degreeCelsius * 9) / 5 + 32;
+const temperatureInFahrenheit = temperature =>
+  console.log(`It is currently ${toFahrenheit(temperature)}ºF at Mars`);
+const greet = temperature =>
+  console.log(
+    `Hi there! Curiosity here. Right now is ${temperature}ºC at Mars`
+  );
+
+const handleError = errorReason =>
+  console.log(`Error getting temperature: ${errorReason}`);
+// definição da função sendMarsTemperature...
+
+const sendMarsTemperature = (callback, error) => {
+  setTimeout(() => {
+    const didOperationSucceed = Math.random() >= 0.6;
```

Comment:
Aqui teria que ser >= 0.4 Pedrão!! desse jeito a gente so tem 40% de chance de enviar a mensagem de volta pra Terra xD
```suggestion
    const didOperationSucceed = Math.random() >= 0.4;
```